Dockerfile.rhel9

```diff
@@ -7,25 +7,21 @@ FROM registry.ci.openshift.org/ocp/4.16:base-rhel9
 COPY --from=builder /go/src/github.com/openshift/cluster-node-tuning-operator/_output/cluster-node-tuning-operator /usr/bin/
 COPY --from=builder /go/src/github.com/openshift/cluster-node-tuning-operator/_output/performance-profile-creator /usr/bin/
 COPY --from=builder /go/src/github.com/openshift/cluster-node-tuning-operator/_output/gather-sysinfo /usr/bin/
+
+ENV ASSETS_DIR=/root/assets
+COPY assets $ASSETS_DIR
+
+COPY hack/dockerfile_install_support.sh /tmp
+RUN /bin/bash /tmp/dockerfile_install_support.sh
+
 COPY manifests/*.yaml manifests/image-references /manifests/
 ENV APP_ROOT=/var/lib/ocp-tuned
 ENV PATH=${APP_ROOT}/bin:${PATH}
 ENV HOME=${APP_ROOT}
 ENV SYSTEMD_IGNORE_CHROOT=1
 WORKDIR ${APP_ROOT}
-COPY assets/bin /usr/local/bin
-RUN INSTALL_PKGS=" \
-      tuned tuned-profiles-atomic tuned-profiles-cpu-partitioning tuned-profiles-mssql tuned-profiles-nfv tuned-profiles-nfv-guest \
-      tuned-profiles-nfv-host tuned-profiles-openshift tuned-profiles-oracle tuned-profiles-postgresql tuned-profiles-realtime \
-      tuned-profiles-sap tuned-profiles-sap-hana tuned-profiles-spectrumscale \
-      nmap-ncat procps-ng pciutils" && \
-    mkdir -p /etc/grub.d/ /boot /var/lib/ocp-tuned && \
-    dnf install --setopt=tsflags=nodocs -y $INSTALL_PKGS && \
-    rm -rf /etc/tuned/recommend.d && \
-    echo auto > /etc/tuned/profile_mode && \
-    sed -Ei 's|^#?\s*enable_unix_socket\s*=.*$|enable_unix_socket = 1|;s|^#?\s*rollback\s*=.*$|rollback = not_on_exit|;s|^#?\s*profile_dirs\s*=.*$|profile_dirs = /var/lib/ocp-tuned/profiles,/usr/lib/tuned|' \
-      /etc/tuned/tuned-main.conf && \
-    touch /etc/sysctl.conf && \
+
+RUN touch /etc/sysctl.conf && \
```

Comment:
As above, would it make sense to move this to `dockerfile_install_support.sh`?

Reply:
Moved to support script.